.github/workflows/reline.yml

```diff
@@ -10,7 +10,7 @@ jobs:
     strategy:
       matrix:
         os: [ ubuntu-latest, macos-latest, windows-latest ]
-        ruby: [ 3.1, '3.0', 2.7, 2.6, 2.5 ]
+        ruby: [ 3.1, 3.0, 2.7, 2.6 ]
```

Comment:
`3.0` needs single-quote because yaml will parse `3.0` to `3`.

Reply:
I didn't know that. Thank you for reviewing!